### PR TITLE
Google Maps in Email will show Address. Fixes #45

### LINKED ISSUE #45: Google Maps just shows "Array" in email notificatoin
image001

### PR DESCRIPTION
**Pull request by avonville**
Email notification sends the array of data for Google Maps integration. Fixed to send the address from the google maps data. Currently, there is no way to add the map to an email. 

We could use the Map Static API in future enhancements, but this will need more testing and research.

Or we can have the google maps section of the email have a link to the entry within weforms.

### CODE REVIEW

File: includes/class-notification.php [hunk 1]
```diff
@@ -661,6 +661,8 @@ public function replace_all_fields( $text = '' ) {
                 } else {
                     $table .= '&mdash;';
                 }
+            }elseif (in_array( $value['type'], [ 'google_map' ] ) ) {
```
Comment (bwmarkle) [inline includes/class-notification.php:664]: Spacing. Change:
`}elseif (in_array( $value['type'], [ 'google_map' ] ) ) {`
to:
`} elseif ( in_array( $value['type'], ['google_map'] ) ) {`

Please see [space usage](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#space-usage).

----

[Use long array syntax](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#declaring-arrays). Change `[ 'google_map' ]` to `array( 'google_map' )`

Comment (bwmarkle) [reply includes/class-notification.php:664]: image

**Review by bwmarkle — CHANGES_REQUESTED** [review]

**Comment by avonville** [comment]
@bwmarkle added the requested changes

**Comment by bwmarkle** [comment]
Hi @avonville please review my comments on this one again. I don't believe they were all addressed.

**Comment by avonville** [comment]
@bwmarkle fixed spacing issues.

**Review by bwmarkle — APPROVED** [review]

**Pull request merged**